src/systems/filecoin_markets/retrieval_market/retrieval_peer_resolver.md

```diff
@@ -0,0 +1,9 @@
+---
+title: "Retrieval Peer Resolver"
+---
+
+The `peer resolver` is a content routing interface to discover retrieval miners that have a given Piece.
```

Comment:
Is this straightforward to build from this description alone?

Reply:
it's a bit of an abstract interface -- it's not exactly mean to describe an implementation, because the primary functionality needs to be supplied by the chain node implementer -- there needs to be an API of the sort "FindDealsOnChainByPieceID".

Also this is a fairly undeveloped piece and I just didn't want to flesh it out any more till I had more knowledge of what it will look like.